docs/search/solr_search_engine.md

```diff
@@ -519,6 +519,19 @@ Connect to the Solr slave interface (http://localhost:8983/solr), go to your cor
 
 ![Solr Slave](solr.png)
 
+## Configuring HTTP Client for Solr queries
+
+Solr uses Symfony HTTP Client to fetch and update Solr index. It's possible to configure timeout and number of max retries for that client using Solr Bundle's Semantic configuration:
```

Comment:
Let's be here more specific
```suggestion
Ibexa Solr Bundle uses Symfony HTTP Client to fetch and update Solr index. It's possible to configure timeout and number of max retries for that client using Solr Bundle's Semantic configuration:
```
because Solr itself is a 3rd party app written in Java, it's our Ibexa integration/client which uses SF HTTP Client.